# Order message
# This structure contains the information of an order.

# ID of the order
string order_id

# Collection of shipments
Shipment[] shipments
